scripts/pylib/twister/twisterlib.py

```diff
@@ -2715,6 +2721,13 @@ def run(self):
             if instance.handler.type_str == "device":
                 instance.handler.suite = self.suite
 
+            if(instance.platform.type == "native"):
+                self.parse_generated()
+                if('CONFIG_ZTEST_DO_THE_SHUFFLE' in self.defconfig and
```

Comment:
Why only propagate the --seed option if the testcase shuffling is selected?

Reply:
I guess you could check for `CONFIG_FAKE_ENTROPY_NATIVE_POSIX`?
(and it could be up to the user to pass the seed for some other reason(?))